src/components/MenuSelect.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Template from './Template';
+import autoHideContainerHOC from '../decorators/autoHideContainer.js';
+import headerFooterHOC from '../decorators/headerFooter.js';
+
+class MenuSelect extends Component {
+  static propTypes = {
+    cssClasses: PropTypes.shape({
+      select: PropTypes.string,
+      option: PropTypes.string,
+    }),
+    items: PropTypes.array.isRequired,
+    refine: PropTypes.func.isRequired,
+    templateProps: PropTypes.object.isRequired,
+  };
+
+  get selectValue() {
```

Comment:
isn't it selectedValue?

Reply:
In my head it was the value of the <select> DOM element, so `selectValue`. But it works the other way around also.